src/app/services/pages.service.ts

```diff
@@ -4,17 +4,17 @@ import { Injectable } from '@angular/core';
   providedIn: 'root'
 })
 export class PagesService {
-
+  window:any = window;
```

Comment:
Is this necessary? Window is already global, so it seems unnecessary to have an instance which points to it.

Reply:
I placed a comment over a test related to this change explaining this. If you want me to reset it and remove the test, I will do so.

Reply:
It appears the service only serves the purpose of hosting a centralized means of refreshing the page with the common window.location.reload. Since the window cannot be spied on and there is no other logic or functionality in the service, I reset everything per your request and simply removed the service from code coverage. This comment also applies to the previous comment from you and Carolyn regarding asana, associated with a comment left in the complement spec file.